src/back/CodeGen/Expr.hs

```diff
@@ -1019,26 +1020,38 @@ instance Translatable A.Expr (State Ctx.Context (CCode Lval, CCode Stat)) where
         callTheMethodOneway
           ntarget targetType name args typeArguments Ty.unitType
 
-      let nullCheck = targetNullCheck ntarget target name emeta "."
+      let nullCheck = targetNullCheck ntarget target name emeta "!"
```

Comment:
"!" is used for my own test, but I forgot to recover before pushing the code. It has been fixed in the next commit.